### PR TITLE
Fix incorrect system cache pod clear command

### PR DESCRIPTION
**Pull request by mikehardy**
with sincere apologies @pmadruga  for the lack of quality in my original PR, this is a lesson in "even when it looks trivial you really need to test it...", there was one more issue in the original PR

I swear it works now! (I actually used the `dev-init` / `dev-sync` tasks to prove it to myself this time)

Should be `pod cache clean --all` not `pod cache clear --all`

Related to #51

One thing worth mentioning is that `executeTask()` can reject, but in all the calls to it except the one for the brew update / brew upgrade pair, the possible rejection is not handled with a `.catch()` on `executeTask` - I think that will fast-fail the script in the future but for now node is just really noisy about how rejections should be handled

That's a bigger change though, maybe an `executeTaskSimple` wrapper that relies on the underlying success/fail reporting and assumes the caller never wants a rejection so eats them all, then call that most of the time?

### CODE REVIEW

File: source/internals/options.js [hunk 1]
```diff
@@ -90,8 +90,8 @@ const askiOSPods = () =>
 
 const askSystemiOSPodsCache = () =>
   new Promise(resolve => {
-    if (args.includes('--remove-system-iOS-pods-cache')) {
-      wipeSystemiOSPodsCache = true;
+    if (args.includes('--keep-system-iOS-pods-cache')) {
```
Comment (pmadruga) [inline source/internals/options.js:93]: Why the change from `remove` to `keep`?

Comment (mikehardy) [reply source/internals/options.js:93]: Sorry - it was in the commit message - I only noticed it yesterday as I was more fully integrating the new version into my workflow / doing release builds 

```
fix: flip pod cache removal arguments from "--remove" to "--keep"

They are in clean-project-auto and default to true per feedback on the feature request,
so then the previous argument style left no way to actually disable them by the command line
if you wanted to preserve your local pod caches

Now you can preserve them on the command line if you like
```

I *think* this mirrors the behavior of other tasks that are default-on and in auto mode, the flag in those is I think to keep things
